block-core/src/transaction.rs

```diff
@@ -34,16 +48,60 @@ impl Encodable for TransactionAction {
             &TransactionAction::Create => {
                 s.encoder().encode_value(&[])
             },
+            &TransactionAction::Create2(salt, ref code) => {
+                s.begin_list(3)
+                    .append(&0xc2_u8)
```

Comment:
That's a descriptor value, I've added it in case we'd need to add more enum variants in TransactionAction. 

Withoug it it would be impossible to add more variants in the future without breaking compatibility for the original Call and Create.

Reply:
Though, is TransactionAction encoding standardized?